src/app/product/Product.module.scss

```
@@ -0,0 +1,236 @@
+@import '../variables.scss';
+
+.container {
+    min-height: 50vh;
+    display: flex;
+    flex-direction: column;
+    justify-content: start;
+    padding: 44px 0;
+    @include responsive-padding;
+}
+
+.card {
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    @media (max-width: 950px) {
+        flex-direction: column;
+    }
+}
+
+.card > * {
+    width: 50%;
+    @media (max-width: 950px) {
+        width: 100%;
+    }
+}
```

```suggestion
.card > * {
    width: 50%;
    
    @media (max-width: 950px) {
        width: 100%;
    }
}
```
Везде добавить отступы перед media.